docs/source/usage/caravel/index.md

```diff
@@ -1516,7 +1516,7 @@ There are around 260 antenna violations with ratios up to 7.
 ⋮
 ```
 
-We can fix those the same way we did in the AES [here](openroad-checkantennas).
+We can fix those the same way we did in the AES [here](#openroadcheckantennas).
```

Comment:
Not the right reference.

Reply:
I tried it, it worked, can you please confirm this is broken?

Reply:
It is not broken. It is jumping to a different section than the one in master.

Reply:
I added explicit section links for the intended sections.